packages/strapi-plugin-content-manager/admin/src/containers/EditPage/index.js

```diff
@@ -334,6 +334,36 @@ export class EditPage extends React.Component {
 
   toggle = () => this.setState(prevState => ({ showWarning: !prevState.showWarning }));
 
+  // After Clicling on a Create Button it jump to the Related
+  addRelatedElement = (relation) => {
+
+    const target = {
+      name: `record.${relation.via}`,
+      value: (relation.nature == 'oneToMany' ? this.props.editPage.record: [this.props.editPage.record]),
+      type: 'select'
+    };
+
+    this.props.match.params = {
+      slug: relation.collection,
+      id: 'create'
+    };
+
+    this.props.resetProps();
+    this.componentDidMount();
```

Comment:
I know that this is a WIP but we have recoded the `content-manager` plugin in order to prepare its extensibility (allowing to add custom code without modifying the core one).
Therefore, we start to follow  conventions and tried to use as much functional programming as possible.

As the one responsible for the front-end I can't allow `this.componentDidMount()` in the codebase, you should either use the `componentDidUpdate()` lifecycle or the sagas.